File: lotteryticket/src/main/java/com/wuhai/lotteryticket/model/interfaces/ILotteryModel.java
package com.wuhai.lotteryticket.model.interfaces;


import com.wuhai.lotteryticket.model.bean.Lottery;

import java.util.List;

/**
 * 作者 wh
 * 
 * 创建日期 2020/2/14 17:53
 * 
 * 描述：
 */
public interface ILotteryModel extends IBaseModelInterFace   {

    /**
     * 彩票
     * @param lottery
     */
    void addLottery(Lottery lottery);

    /**
     * 查询所有数据
     * @return
     */
    List<Lottery> queryLotteryAll();
}

File: lotteryticketkotlin/src/main/java/com/wuhai/demo/lotteryticketkotlin/model/interfaces/ILotteryModel.kt
package com.wuhai.demo.lotteryticketkotlin.model.interfaces

import com.wuhai.demo.lotteryticketkotlin.model.bean.Lottery

/**
 * 作者 wh
 *
 * 创建日期 2020/2/14 17:53
 *
 * 描述：
 */
interface ILotteryModel : IBaseModelInterFace {
    /**
     * 彩票
     * @param lottery
     */
    fun addLottery(lottery: Lottery?)

    /**
     * 查询所有数据
     * @return
     */
    fun queryLotteryAll(): List<Lottery?>?

    /**
     * 删除
     * @param lottery
     */
    fun deleteLottery(lottery: Lottery?)
}